README.md

```diff
@@ -153,6 +154,18 @@ It will look like the following:
 
 > NOTE: It may take 5-10 minutes after you see 'SUCCESS' for the application to be fully deployed. If you see a "Python Developer" welcome screen or an error page, then wait a bit and refresh the page. See [guide on debugging App Service deployments](docs/appservice.md).
 
+### Deploying to Azure Container Apps
```

Comment:
I'm trying to reduce the size of this README and not add any more headings. 

How about changing this line here:

"Follow these steps to provision Azure resources and deploy the application code:"

"The steps below will provision Azure resources and deploy the application code to Azure App Service. To deploy to Azure Container Apps instead, follow [the container apps deployment guide](docs/azure_container_apps.md).
"

Reply:
Done.